pkgs/tools/misc/rtx/default.nix

```diff
@@ -1,16 +1,18 @@
-{ lib
-, nix-update-script
-, rustPlatform
+{ Security
```

Comment:
Undo everything unrelated to the title of your PR.

Reply:
see the current changes